### PR TITLE
Add design tokens

### PR DESCRIPTION
**Pull request by dbenmore**
https://github.com/ConnectedHomes/nucleus/issues/1890

### CODE REVIEW

File: docs/getting-started/design-tokens.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## What are design tokens?
+
+Design tokens are a technology agnostic way to create and share reusable variables. They open up a number of possibilities for Nucleus including supporting technologies beyond the web, enabling multiple brand identities and creating a shared design language across the business.
+
+## Why use design tokens in your product
+
+Every brand has design details that are core to the brand’s identity. These details need to persist across multiple channels and technologies to ensure the brand is easily recognisable and create a consistent brand experience.
+
+Design details are not limited to design foundations such as colours and fonts, component design also plays a part in brand recognition. A common example is the brands primary button style that gets reproduced on websites, apps, emails, social posts, tv adverts and the sides of busses!
```
Comment (RobTobias123) [inline docs/getting-started/design-tokens.md:15]: test

**Review by RobTobias123 — APPROVED** [review]

**Review by PtitBen — COMMENTED** [review]
Found a few things, some of them might just be my english ;-)

File: docs/getting-started/design-tokens.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    "color": { brand.colour.secondary.value },
+    "width": { brand.border.width.value },
+    "style": { brand.border.style.value }
+  }
+}
+```
+
+## Supported technologies
+
+The current technologies we supply design tokens for support are:
```
Comment (PtitBen) [inline docs/getting-started/design-tokens.md:116]: > The current technologies we supply design tokens for support are:

Maybe just me, if it is, ignore.
I would phrase it like this: 

_We supply design token support for the following technologies:_

File: docs/getting-started/design-tokens.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+title: Design tokens
+description: Small and mighty
+---
+
+
+## What are design tokens?
+
+Design tokens are a technology agnostic way to create and share reusable variables. They open up a number of possibilities for Nucleus including supporting technologies beyond the web, enabling multiple brand identities and creating a shared design language across the business.
+
+## Why use design tokens in your product
```
[the remark this answers is not in the document]
Comment (dbenmore) [reply docs/getting-started/design-tokens.md]: Done

File: docs/getting-started/design-tokens.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+Every brand has design details that are core to the brand’s identity. These details need to persist across multiple channels and technologies to ensure the brand is easily recognisable and create a consistent brand experience.
+
+Design details are not limited to design foundations such as colours and fonts, component design also plays a part in brand recognition. A common example is the brands primary button style that gets reproduced on websites, apps, emails, social posts, tv adverts and the sides of busses!
+
+Without design tokens each technology has to maintain it's own set of variables and keep pace with the latest brand identity guidelines. Design debt often creeps in and variations in the brand occur.
+
+Utilising design tokens in your product allows Nucleus to accurately share design decisions with you through a common set of technology specific variables. Futhermore, consistent naming conventions allows nucleus to send you the latest brand changes and reduce the effort needed to implement brand updates. For example if the brand wants to change the font used for all headings we can update the tokens centrally and you would only need to pull the latest version.
```
[the remark this answers is not in the document]
Comment (dbenmore) [reply docs/getting-started/design-tokens.md]: Done

File: docs/getting-started/design-tokens.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+## How they work
+
+The tokens are stored in JSON format and then passed through a tool that converts them in to language specific variables that can be used in a variety of technologies. We've chosen Style Dictionary to help us manage design tokens.
+
+![Design token creation diagram](https://user-images.githubusercontent.com/10724413/107638686-d2437680-6c67-11eb-9630-b82add6ee716.png)
+
+### Component specifications
+
+To provide a clear guide on how to create a component we have created a named subset of tokens for each component. This means that consumers of the tokens will only need to ensure that are using them in the right place not specific the values of the tokens. Taking this approach leverages one of the more powerful aspects of design tokens as it allows us to update components centrally and enables multibranding functionality.
```
[the remark this answers is not in the document]
Comment (dbenmore) [reply docs/getting-started/design-tokens.md]: Agreed it didn't make sense. I've re-worded it so hopefully it's better now?

File: docs/getting-started/design-tokens.md (hunk 2 above)
Comment (dbenmore) [reply docs/getting-started/design-tokens.md]: Changed to "We supply design tokens for the following technologies:"

**Review by andij — APPROVED** [review]

**Pull request merged**